pom.xml

```diff
@@ -412,6 +412,27 @@
                 <version>${mail.version}</version>
             </dependency>
 
+            <dependency>
+                <groupId>org.apache.curator</groupId>
+                <artifactId>curator-framework</artifactId>
```

Comment:
add this curator in license file

Reply:
What's the file name? I found a `LICENSE` file in project root folder but seems not it.

Reply:
resolved.